pkg/tinyusb/contrib/tinyusb_descriptors.c

```diff
@@ -23,6 +23,8 @@
  * THE SOFTWARE.
  */
 
+#define USB_H_USER_IS_RIOT_INTERNAL
+
```

Comment:
Out of the scope but why is this needed ?

Reply:
Yes, if the user doesn`t define `CONFIG_USB_VID` and `CONFIG_USB_PID`, the default RIOT peripheral VID/PID `1209:7d00` is used only if `USB_H_USER_IS_RIOT_INTERNAL` is defined when [sys/include/usb.h](https://github.com/RIOT-OS/RIOT/blob/master/sys/include/usb.h) is included. Otherwise the compilation fails with the error message because `CONFIG_USB_VID` and `CONFIG_USB_PID` are used for the device descriptor: https://github.com/RIOT-OS/RIOT/blob/df0c04ecc83ca246c2fec7566b8c6c13b56f8d0f/sys/include/usb.h#L48-L56
We didn't detect this problem before using tinyUSB in applications other than the tests, because the tests use the test VID/PID.

Reply:
Also the DFU bootloader sets the VID/PID pair.